apps/desktop/src/lib/components/EditMode.svelte

```diff
@@ -14,16 +18,95 @@
 	const { editModeMetadata }: Props = $props();
 
 	const project = getContext(Project);
+	const uncommitedFileWatcher = getContext(UncommitedFilesWatcher);
 	const modeService = getContext(ModeService);
 
+	const uncommitedFiles = uncommitedFileWatcher.uncommitedFiles;
+
+	let modeServiceAborting = $state<'inert' | 'loading' | 'completed'>('inert');
```

Comment:
Nit pick: I'd love these to be extracted to an enum / as const object, same with the `outputMap.status` field, instead of "magic" strings of `'D', 'M', 'A'`, ya know